### ISSUE TITLE
Feature Suggestions - Tab Count, Window Naming, Window Fold + Expand

### ISSUE THREAD

**Issue by Cr8zyIvan** [Bill13579/tabby#206]
**Tab Count**
I would find it useful to see the number of Tabs per Window. Perhaps something like a badge number on the right side of the Window Title (Title bar?). I could see a "fractional number", where the number on top would be the order number of the active tab, and the number on the bottom would be the total number of tabs in that Window. Something like "4/10" would mean that the Active Tab is the 4th Tab in that Window, and there are 10 Tabs in that Window.

**Window Naming**
The Window Names are indicated as "Window 1", "Window 2", etc. As personal preference, I would like the Window Name to become the Title of it's Active Tab. Perhaps there could be an Toggle in the Options where the user can choose to leave Window Naming as it currently is or choose to have each Window adopt it's Active Tab's Name. Even though not perfect, I would still find it easier to know to which Window I want to transfer my selected tabs (using the Context Menu), instead of seeing "Window 1", etc., and not knowing exactly which Window that is.

**Window Fold + Expand**
In the side bar, I would find it useful to be able to Fold / Expand each Window to visualize all the Windows I have open. Just an idea that comes to mind, I can see an option where there would be a "partial Expand", where you could see 1\~2\~3 Tabs Expanded under each Window, accompanied with a "..." symbol (3 dots) to mean there are still more Tabs.

In any case, I find Tabby very useful and use it almost every day. Cheers!

**Comment by BJTorgs** [comment]
+1

**Comment by riccrastle** [comment]
Not sure what you mean by the name being the title of its active tab, but maybe that has something to do with your OS or your window manager. I'm using XFCE, which seems to do what you're suggesting by default.
![image](https://github.com/user-attachments/assets/8a237c82-22bb-466a-9b02-d700b944acbf)

I do want to add a +1 to folding the windows. At the very least, I think it would make navigating the add-on's drop-down easier. I'd like to rename windows without having to scroll past the other windows' tab lists.

**Comment by nigz891** [comment]
Yeah, tab counter on the icon and the ability to create new tab would make this useful.  I could get rid of two buttons from my toolbar and still be able to go tab-bar free.
